targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/mbed_crc_api.c

```diff
@@ -0,0 +1,101 @@
+#include "crc_api.h"
+
+#include "drivers/fsl_crc.h"
+#include "platform/mbed_assert.h"
+
+#ifdef DEVICE_CRC
+
+static crc_bits_t width;
+
+bool hal_crc_is_supported(const uint32_t polynomial)
+{
+  switch (polynomial)
+  {
+    case POLY_16BIT_CCITT:
+    case POLY_16BIT_IBM:
+    case POLY_32BIT_ANSI:
+      return true;
```

Comment:
K64F supports only specified polynomials or any polynomial with width 16/32?
If we have  hal API to set configs, user can select any polynomial and not just above three.